src/System.Net.Http/ref/project.json

```diff
@@ -3,14 +3,15 @@
     "System.Runtime": "4.0.0",
     "System.Threading.Tasks": "4.0.0",
     "System.IO": "4.0.0",
-    "System.Net.Primitives": "4.0.0",
+    "System.Net.Primitives": "4.0.10",
+    "System.Security.Cryptography.X509Certificates": "4.1.0-rc3-24008-00",
```

Comment:
This is going to cause Crypto to be pulled into standard library (fyi @bartonjs), I'm not necessarily against it but I want others to be aware of it and make sure we are intentional about it.

cc @KrzysztofCwalina @terrajobst